### DISCUSSION TITLE
Navier Stokes Module in Combined Module

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#26235] [Q&A Modules: General]
I have found that when I used the combined app (freshly installed MOOSE) that there are postprocessors from the Navier Stokes app that are not available for use in the combined app. Is there a way for me to point the combined app to the directory in the NS app that has the required postprocessors? I have found that copying the .h and .C files to a folder used by the combined app allows me use of the post processors. Some of the postprocessors are too convoluted and connected to multiple other files to use a copying method to use them.

**Comment by GiudGiud** [comment]
Hello

The combined app should include NS, moving the files should not be necessary. 
Are you on linux or macos? Can you ldd (linux) / otool -L (macos) the combined executable please?

Guillaume

**Reply by TLWise** [reply]
Is this the correct syntax for the information that you requested? ~/projects/moose/modules/combined$ ldd /bin/ls

**Reply by GiudGiud** [reply]
`ldd combined-opt` 
in that same directory

**Reply by TLWise** [reply]
Lots of information follows. Thanks for taking a look.:

	linux-vdso.so.1 (0x00007ffc623fd000)
	libcombined_test-opt.so.0 => /home/twise/projects/moose/modules/combined/test/lib/libcombined_test-opt.so.0 (0x00007f218eab9000)
	libcombined-opt.so.0 => /home/twise/projects/moose/modules/combined/lib/libcombined-opt.so.0 (0x00007f218eaa9000)
	libxfem-opt.so.0 => /home/twise/projects/moose/modules/xfem/lib/libxfem-opt.so.0 (0x00007f218e600000)
	libscalar_transport-opt.so.0 => /home/twise/projects/moose/modules/scalar_transport/lib/libscalar_transport-opt.so.0 (0x00007f218ea32000)
	libporous_flow-opt.so.0 => /home/twise/projects/moose/modules/porous_flow/lib/libporous_flow-opt.so.0 (0x00007f218e000000)
	libphase_field-opt.so.0 => /home/twise/projects/moose/modules/phase_field/lib/libphase_field-opt.so.0 (0x00007f218d600000)
	libperidynamics-opt.so.0 => /home/twise/projects/moose/modules/peridynamics/lib/libperidynamics-opt.so.0 (0x00007f218d200000)
	liboptimization-opt.so.0 => /home/twise/projects/moose/modules/optimization/lib/liboptimization-opt.so.0 (0x00007f218d021000)
	libfsi-opt.so.0 => /home/twise/projects/moose/modules/fsi/lib/libfsi-opt.so.0 (0x00007f218e9a3000)
	libcontact-opt.so.0 => /home/twise/projects/moose/modules/contact/lib/libcontact-opt.so.0 (0x00007f218cc00000)
	libstochastic_tools-opt.so.0 => /home/twise/projects/moose/modules/stochastic_tools/lib/libstochastic_tools-opt.so.0 (0x00007f218c400000)
	librichards-opt.so.0 => /home/twise/projects/moose/modules/richards/lib/librichards-opt.so.0 (0x00007f218c000000)
	libreactor-opt.so.0 => /home/twise/projects/moose/modules/reactor/lib/libreactor-opt.so.0 (0x00007f218d43e000)
	liblevel_set-opt.so.0 => /home/twise/projects/moose/modules/level_set/lib/liblevel_set-opt.so.0 (0x00007f218e8d1000)
	libgeochemistry-opt.so.0 => /home/twise/projects/moose/modules/geochemistry/lib/libgeochemistry-opt.so.0 (0x00007f218ce61000)
	libfunctional_expansion_tools-opt.so.0 => /home/twise/projects/moose/modules/functional_expansion_tools/lib/libfunctional_expansion_tools-opt.so.0 (0x00007f218df34000)
	libexternal_petsc_solver-opt.so.0 => /home/twise/projects/moose/modules/external_petsc_solver/lib/libexternal_petsc_solver-opt.so.0 (0x00007f218def9000)
	libelectromagnetics-opt.so.0 => /home/twise/projects/moose/modules/electromagnetics/lib/libelectromagnetics-opt.so.0 (0x00007f218cb2c000)
	libmoose-opt.so.0 => /home/twise/projects/moose/framework/libmoose-opt.so.0 (0x00007f2189600000)
	libpcre-opt.so.0 => /home/twise/projects/moose/framework/contrib/pcre/libpcre-opt.so.0 (0x00007f218decc000)
	libhit-opt.so.0 => /home/twise/projects/moose/framework/contrib/hit/libhit-opt.so.0 (0x00007f218c392000)
	libchemical_reactions_test-opt.so.0 => /home/twise/projects/moose/modules/chemical_reactions/test/lib/libchemical_reactions_test-opt.so.0 (0x00007f218e5ee000)
	libelectromagnetics_test-opt.so.0 => /home/twise/projects/moose/modules/electromagnetics/test/lib/libelectromagnetics_test-opt.so.0 (0x00007f218ce2b000)
	libexternal_petsc_solver_test-opt.so.0 => /home/twise/projects/moose/modules/external_petsc_solver/test/lib/libexternal_petsc_solver_test-opt.so.0 (0x00007f218debd000)
	libfluid_properties_test-opt.so.0 => /home/twise/projects/moose/modules/fluid_properties/test/lib/libfluid_properties_test-opt.so.0 (0x00007f218c2ef000)
	libfunctional_expansion_tools_test-opt.so.0 => /home/twise/projects/moose/modules/functional_expansion_tools/test/lib/libfunctional_expansion_tools_test-opt.so.0 (0x00007f218cb1c000)
	libgeochemistry_test-opt.so.0 => /home/twise/projects/moose/modules/geochemistry/test/lib/libgeochemistry_test-opt.so.0 (0x00007f218cb0d000)
	liblevel_set_test-opt.so.0 => /home/twise/projects/moose/modules/level_set/test/lib/liblevel_set_test-opt.so.0 (0x00007f218cafe000)
	libray_tracing_test-opt.so.0 => /home/twise/projects/moose/modules/ray_tracing/test/lib/libray_tracing_test-opt.so.0 (0x00007f21894d2000)
	librdg_test-opt.so.0 => /home/twise/projects/moose/modules/rdg/test/lib/librdg_test-opt.so.0 (0x00007f218c2cf000)
	libreactor_test-opt.so.0 => /home/twise/projects/moose/modules/reactor/test/lib/libreactor_test-opt.so.0 (0x00007f218c2c0000)
	librichards_test-opt.so.0 => /home/twise/projects/moose/modules/richards/test/lib/librichards_test-opt.so.0 (0x00007f218c2af000)
	libstochastic_tools_test-opt.so.0 => /home/twise/projects/moose/modules/stochastic_tools/test/lib/libstochastic_tools_test-opt.so.0 (0x00007f21893e7000)
	libheat_transfer_test-opt.so.0 => /home/twise/projects/moose/modules/heat_transfer/test/lib/libheat_transfer_test-opt.so.0 (0x00007f218bfb6000)
	libsolid_properties_test-opt.so.0 => /home/twise/projects/moose/modules/solid_properties/test/lib/libsolid_properties_test-opt.so.0 (0x00007f21893b9000)
	libtensor_mechanics_test-opt.so.0 => /home/twise/projects/moose/modules/tensor_mechanics/test/lib/libtensor_mechanics_test-opt.so.0 (0x00007f2189261000)
	libnavier_stokes_test-opt.so.0 => /home/twise/projects/moose/modules/navier_stokes/test/lib/libnavier_stokes_test-opt.so.0 (0x00007f2189113000)
	libcontact_test-opt.so.0 => /home/twise/projects/moose/modules/contact/test/lib/libcontact_test-opt.so.0 (0x00007f218c28c000)
	libfsi_test-opt.so.0 => /home/twise/projects/moose/modules/fsi/test/lib/libfsi_test-opt.so.0 (0x00007f218bfa7000)
	libmisc_test-opt.so.0 => /home/twise/projects/moose/modules/misc/test/lib/libmisc_test-opt.so.0 (0x00007f21890c1000)
	liboptimization_test-opt.so.0 => /home/twise/projects/moose/modules/optimization/test/lib/liboptimization_test-opt.so.0 (0x00007f218903e000)
	libperidynamics_test-opt.so.0 => /home/twise/projects/moose/modules/peridynamics/test/lib/libperidynamics_test-opt.so.0 (0x00007f2188fe9000)
	libphase_field_test-opt.so.0 => /home/twise/projects/moose/modules/phase_field/test/lib/libphase_field_test-opt.so.0 (0x00007f2188f65000)
	libporous_flow_test-opt.so.0 => /home/twise/projects/moose/modules/porous_flow/test/lib/libporous_flow_test-opt.so.0 (0x00007f2188f19000)
	libthermal_hydraulics_test-opt.so.0 => /home/twise/projects/moose/modules/thermal_hydraulics/test/lib/libthermal_hydraulics_test-opt.so.0 (0x00007f2188d81000)
	libscalar_transport_test-opt.so.0 => /home/twise/projects/moose/modules/scalar_transport/test/lib/libscalar_transport_test-opt.so.0 (0x00007f2188d72000)
	libchemical_reactions-opt.so.0 => /home/twise/projects/moose/modules/chemical_reactions/lib/libchemical_reactions-opt.so.0 (0x00007f2188c46000)
	libthermal_hydraulics-opt.so.0 => /home/twise/projects/moose/modules/thermal_hydraulics/lib/libthermal_hydraulics-opt.so.0 (0x00007f2188200000)
	libnavier_stokes-opt.so.0 => /home/twise/projects/moose/modules/navier_stokes/lib/libnavier_stokes-opt.so.0 (0x00007f2187800000)
	libfluid_properties-opt.so.0 => /home/twise/projects/moose/modules/fluid_properties/lib/libfluid_properties-opt.so.0 (0x00007f2187400000)
	librdg-opt.so.0 => /home/twise/projects/moose/modules/rdg/lib/librdg-opt.so.0 (0x00007f2188b8e000)
	libsolid_properties-opt.so.0 => /home/twise/projects/moose/modules/solid_properties/lib/libsolid_properties-opt.so.0 (0x00007f2187743000)
	libheat_transfer-opt.so.0 => /home/twise/projects/moose/modules/heat_transfer/lib/libheat_transfer-opt.so.0 (0x00007f2187000000)
	libray_tracing-opt.so.0 => /home/twise/projects/moose/modules/ray_tracing/lib/libray_tracing-opt.so.0 (0x00007f2186c00000)
	libmisc-opt.so.0 => /home/twise/projects/moose/modules/misc/lib/libmisc-opt.so.0 (0x00007f2186f6e000)
	libxfem_test-opt.so.0 => /home/twise/projects/moose/modules/xfem/test/lib/libxfem_test-opt.so.0 (0x00007f21876f1000)
	libtensor_mechanics-opt.so.0 => /home/twise/projects/moose/modules/tensor_mechanics/lib/libtensor_mechanics-opt.so.0 (0x00007f2185c00000)
	libwaspcore.so.04 => /home/twise/mambaforge3/envs/moose/wasp/lib/libwaspcore.so.04 (0x00007f2188b6d000)
	libwaspddi.so.04 => /home/twise/mambaforge3/envs/moose/wasp/lib/libwaspddi.so.04 (0x00007f21876d2000)
	libwaspexpr.so.04 => /home/twise/mambaforge3/envs/moose/wasp/lib/libwaspexpr.so.04 (0x00007f2186f3a000)
	libwasphalite.so.04 => /home/twise/mambaforge3/envs/moose/wasp/lib/libwasphalite.so.04 (0x00007f2186ee5000)
	libwasphit.so.04 => /home/twise/mambaforge3/envs/moose/wasp/lib/libwasphit.so.04 (0x00007f2186e9e000)
	libwasphive.so.04 => /home/twise/mambaforge3/envs/moose/wasp/lib/libwasphive.so.04 (0x00007f21881f6000)
	libwaspjson.so.04 => /home/twise/mambaforge3/envs/moose/wasp/lib/libwaspjson.so.04 (0x00007f2186e5a000)
	libwasplsp.so.04 => /home/twise/mambaforge3/envs/moose/wasp/lib/libwasplsp.so.04 (0x00007f2186e1d000)
	libwaspsiren.so.04 => /home/twise/mambaforge3/envs/moose/wasp/lib/libwaspsiren.so.04 (0x00007f2186be4000)
	libwaspson.so.04 => /home/twise/mambaforge3/envs/moose/wasp/lib/libwaspson.so.04 (0x00007f2186b96000)
	libmesh_opt.so.0 => /home/twise/mambaforge3/envs/moose/libmesh/lib/libmesh_opt.so.0 (0x00007f2184400000)
	libtimpi_opt.so.11 => /home/twise/mambaforge3/envs/moose/libmesh/lib/libtimpi_opt.so.11 (0x00007f21876c3000)
	libvtkIOCore-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkIOCore-9.2.so.1 (0x00007f2186b28000)
	libvtkCommonCore-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkCommonCore-9.2.so.1 (0x00007f2183a00000)
	libvtkCommonDataModel-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkCommonDataModel-9.2.so.1 (0x00007f2183400000)
	libvtkFiltersCore-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkFiltersCore-9.2.so.1 (0x00007f2182a00000)
	libvtkIOXML-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkIOXML-9.2.so.1 (0x00007f2185ae6000)
	libvtkImagingCore-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkImagingCore-9.2.so.1 (0x00007f2182600000)
	libvtkIOImage-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkIOImage-9.2.so.1 (0x00007f2182422000)
	libvtkImagingMath-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkImagingMath-9.2.so.1 (0x00007f2186adf000)
	libvtkIOParallelXML-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkIOParallelXML-9.2.so.1 (0x00007f2186a96000)
	libvtkParallelMPI-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkParallelMPI-9.2.so.1 (0x00007f2186a7a000)
	libvtkParallelCore-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkParallelCore-9.2.so.1 (0x00007f218439f000)
	libvtkCommonExecutionModel-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtkCommonExecutionModel-9.2.so.1 (0x00007f21842a1000)
	libvtksys-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/libvtksys-9.2.so.1 (0x00007f21839af000)
	libz.so.1 => /home/twise/mambaforge3/envs/moose/lib/libz.so.1 (0x00007f2186a5f000)
	libslepc.so.3.20 => /home/twise/mambaforge3/envs/moose/lib/libslepc.so.3.20 (0x00007f2182000000)
	libpetsc.so.3.20 => /home/twise/mambaforge3/envs/moose/lib/libpetsc.so.3.20 (0x00007f2180400000)
	libHYPRE-2.29.0.so => /home/twise/mambaforge3/envs/moose/lib/libHYPRE-2.29.0.so (0x00007f217fe00000)
	libstrumpack.so.7.1 => /home/twise/mambaforge3/envs/moose/lib/libstrumpack.so.7.1 (0x00007f217ec00000)
	libscalapack.so.2.2 => /home/twise/mambaforge3/envs/moose/lib/libscalapack.so.2.2 (0x00007f217e400000)
	libsuperlu_dist.so.8 => /home/twise/mambaforge3/envs/moose/lib/libsuperlu_dist.so.8 (0x00007f218327c000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f218c287000)
	libparmetis.so => /home/twise/mambaforge3/envs/moose/lib/libparmetis.so (0x00007f2183971000)
	libmetis.so => /home/twise/mambaforge3/envs/moose/lib/libmetis.so (0x00007f2182995000)
	libhdf5_hl.so.200 => /home/twise/mambaforge3/envs/moose/lib/libhdf5_hl.so.200 (0x00007f218427d000)
	libhdf5.so.200 => /home/twise/mambaforge3/envs/moose/lib/libhdf5.so.200 (0x00007f217de00000)
	libmpifort.so.12 => /home/twise/mambaforge3/envs/moose/lib/libmpifort.so.12 (0x00007f2182944000)
	libgfortran.so.5 => /home/twise/mambaforge3/envs/moose/lib/libgfortran.so.5 (0x00007f217fc55000)
	libgcc_s.so.1 => /home/twise/mambaforge3/envs/moose/lib/libgcc_s.so.1 (0x00007f2183956000)
	libquadmath.so.0 => /home/twise/mambaforge3/envs/moose/lib/libquadmath.so.0 (0x00007f218290b000)
	libstdc++.so.6 => /home/twise/mambaforge3/envs/moose/lib/libstdc++.so.6 (0x00007f217dc1d000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007f21873e8000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f2186e18000)
	libmpi.so.12 => /home/twise/mambaforge3/envs/moose/lib/libmpi.so.12 (0x00007f217d800000)
	libmpicxx.so.12 => /home/twise/mambaforge3/envs/moose/lib/libmpicxx.so.12 (0x00007f2183937000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f2181f19000)
	libgomp.so.1 => /home/twise/mambaforge3/envs/moose/lib/libgomp.so.1 (0x00007f21828cf000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f217d400000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f218eafc000)
	libnetcdf.so.13 => /home/twise/mambaforge3/envs/moose/libmesh/lib/./libnetcdf.so.13 (0x00007f21802f0000)
	libvtkdoubleconversion-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkdoubleconversion-9.2.so.1 (0x00007f2185ad5000)
	libvtklz4-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtklz4-9.2.so.1 (0x00007f218289b000)
	libvtklzma-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtklzma-9.2.so.1 (0x00007f21823f5000)
	libvtkzlib-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkzlib-9.2.so.1 (0x00007f218325e000)
	libvtkloguru-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkloguru-9.2.so.1 (0x00007f21823bd000)
	libvtkCommonTransforms-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkCommonTransforms-9.2.so.1 (0x00007f2181ed6000)
	libvtkCommonMisc-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkCommonMisc-9.2.so.1 (0x00007f217ce00000)
	libvtkCommonSystem-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkCommonSystem-9.2.so.1 (0x00007f2181e96000)
	libvtkpugixml-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkpugixml-9.2.so.1 (0x00007f21802b3000)
	libvtkCommonMath-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkCommonMath-9.2.so.1 (0x00007f218028c000)
	libvtkIOXMLParser-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkIOXMLParser-9.2.so.1 (0x00007f2181e7c000)
	libvtkDICOMParser-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkDICOMParser-9.2.so.1 (0x00007f218026f000)
	libvtkmetaio-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkmetaio-9.2.so.1 (0x00007f217eb62000)
	libvtkpng-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkpng-9.2.so.1 (0x00007f217eb1b000)
	libvtktiff-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtktiff-9.2.so.1 (0x00007f217e37e000)
	libvtkjpeg-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkjpeg-9.2.so.1 (0x00007f217e301000)
	libvtkIOLegacy-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/./libvtkIOLegacy-9.2.so.1 (0x00007f217e242000)
	libzfp.so.0 => /home/twise/mambaforge3/envs/moose/lib/./libzfp.so.0 (0x00007f217d7ac000)
	libcrypto.so.1.1 => /home/twise/mambaforge3/envs/moose/lib/./libcrypto.so.1.1 (0x00007f217ca00000)
	libcurl.so.4 => /home/twise/mambaforge3/envs/moose/lib/./libcurl.so.4 (0x00007f217d709000)
	libvtkkissfft-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/././libvtkkissfft-9.2.so.1 (0x00007f2184275000)
	libvtkexpat-9.2.so.1 => /home/twise/mambaforge3/envs/moose/libmesh-vtk/lib/././libvtkexpat-9.2.so.1 (0x00007f217fc22000)
	libnghttp2.so.14 => /home/twise/mambaforge3/envs/moose/lib/././libnghttp2.so.14 (0x00007f217d6dd000)
	libssh2.so.1 => /home/twise/mambaforge3/envs/moose/lib/././libssh2.so.1 (0x00007f217d699000)
	libssl.so.1.1 => /home/twise/mambaforge3/envs/moose/lib/././libssl.so.1.1 (0x00007f217d370000)
	libgssapi_krb5.so.2 => /home/twise/mambaforge3/envs/moose/lib/././libgssapi_krb5.so.2 (0x00007f217d646000)
	libkrb5.so.3 => /home/twise/mambaforge3/envs/moose/lib/./././libkrb5.so.3 (0x00007f217d299000)
	libk5crypto.so.3 => /home/twise/mambaforge3/envs/moose/lib/./././libk5crypto.so.3 (0x00007f217eb01000)
	libcom_err.so.3 => /home/twise/mambaforge3/envs/moose/lib/./././libcom_err.so.3 (0x00007f218426b000)
	libkrb5support.so.0 => /home/twise/mambaforge3/envs/moose/lib/./././libkrb5support.so.0 (0x00007f217e233000)
	libkeyutils.so.1 => /home/twise/mambaforge3/envs/moose/lib/./././libkeyutils.so.1 (0x00007f2182894000)
	libresolv.so.2 => /lib/x86_64-linux-gnu/libresolv.so.2 (0x00007f217d632000)

**Reply by GiudGiud** [reply]
Navier stokes does show up in the list. Can you attach a full syntax dump ? I think you need to add  --json or --dump  after the executable on the command line. You ll have to put it in a text file

**Reply by TLWise** [reply]
Still a rookie at this command line stuff, BCs and ICs are much easier. Should it look like so:
../../combined-opt --dump -i DGF_Benchmark_Scaling_No_HT.i

**Reply by GiudGiud** [reply]
just
```
../../combined-opt --dump
```

**Comment by TLWise** [comment]
There was so much data that it did not all display on the terminal display. Is there a way to have it save to a data file instead? Here is what I was able to copy and paste. [datadump.txt](https://github.com/idaholab/moose/files/13541220/datadump.txt)

**Reply by GiudGiud** [reply]
```
../../combined-opt --dump &> my_file.txt
```

**Reply by TLWise** [reply]
Yep. I was able to figure out the proper syntax to do that. The file is 31 MB the max paste size is 25 MB. I'll chop it down by taking out the last of the file, since you will already have that from the previous paste.

**Comment by TLWise** [comment]
[datadump2_part1.txt](https://github.com/idaholab/moose/files/13541467/datadump2_part1.txt) Whew!

**Reply by GiudGiud** [reply]
thx
Interesting. You see the action, the kernels, the BCs, some PPs from Navier Stokes, but not all the PPs. I dont see volumetricFlowRate for example

**Reply by TLWise** [reply]
Is there a way to resolve that issue without a major modification to the code? I'm open to doing a clean install.

**Reply by GiudGiud** [reply]
clean + re-install is a good idea.
I dont see what could have gone wrong
